tests/eas/load_tracking.py

```diff
@@ -1293,3 +1294,119 @@ def test_util_tg4_group(self):
+    trace = None
+
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
+        # Run all workloads on a CPU with highest capacity
+        cls.target_cpu = min(test_env.calibration(),
+                             key=test_env.calibration().get)
+
+        # Create taskgroups
+        cpus = test_env.target.list_online_cpus()
+        mems = 0
+        cls.root_group = Taskgroup("/", cpus, mems, test_env)
+        tg1 = Taskgroup("/tg1", cpus, mems, test_env)
+        tg2 = Taskgroup("/tg2", cpus, mems, test_env)
+
+        # Create tasks
+        period_ms = 16
+        duration_s = 4
+        cpus = [cls.target_cpu]
+        t1_1 = Task("t1_1", test_env, cpus, period_ms=period_ms,
+                    duty_cycle_pct=10, duration_s=duration_s)
+        t1_2 = Task("t1_2", test_env, cpus, period_ms=period_ms,
+                    duty_cycle_pct=20, duration_s=duration_s)
+        t2_1 = Task("t2_1", test_env, cpus, period_ms=period_ms,
+                    duty_cycle_pct=5, duration_s=duration_s)
+
+        # Link nodes to the hierarchy tree
+        cls.root_group.add_children([tg1, tg2])
+        tg1.add_children([t1_1, t1_2])
+        tg2.add_children([t2_1])
+
+        conf = {
+            'tag' : 'cgp_migration',
+            'flags' : ['ftrace', 'freeze_userspace'],
+            'cpufreq' : {'governor' : 'performance'},
+        }
+
+        return {
+            'wloads': {},
+            'confs': [conf],
+        }
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super(CgroupsMigrationTest, cls).runExperiments(*args, **kwargs)
+
+    @classmethod
+    def _migrate_task(cls, test_env):
+        task = None
+        new_taskgroup = None
+        exclude = ''
+        cgroups = test_env.target.cgroups
+
+        # Get the task and taskgroup for the migration
+        for se in cls.root_group.iter_nodes():
+            if se.name == 't1_2':
+                task = se
+            if se.name == '/tg2':
+                new_taskgroup = se
+
+        # Get the PID of the task that is not migrated
+        tg1_task = cgroups.controllers['cpu'].tasks('/tg1')
+        for pid, data in tg1_task.iteritems():
+            if data[0] == 'rt-app' and re.search(r't1_1', data[1]):
+                exclude = '{}'.format(pid)
+
+        # Migrate t1_2 to the group /tg2 without migrating t1_1
+        task.change_taskgroup(new_taskgroup)
+        srcg = cgroups.controllers['cpu'].cgroup('/tg1').directory
+        dstg = cgroups.controllers['cpu'].cgroup('/tg2').directory
+        cgroups.cgroups_tasks_move(srcg, dstg, exclude=exclude)
+
+    def test_util_root_group(self):
+        """Test utilization propagation to cgroup root"""
+        return self._test_group_util('/')
+
+    def test_util_tg1_group(self):
+        """Test utilization propagation to cgroup /tg1"""
+        return self._test_group_util('/tg1')
+
+    def test_util_tg1_1_group(self):
+        """Test utilization propagation to cgroup /tg2"""
```

Comment:
This comment suggests a different test function name, is that intentional?

Reply:
Oops done :)